pkg/plugins/runtime/k8s/metadata/annotations.go

```diff
@@ -134,6 +134,10 @@ const (
 
 	// HeadlessService is "true" when the Service had ClusterIP: None, otherwise "false"
 	HeadlessService = "k8s.kuma.io/is-headless-service"
+
+	// KumaXdsTransportProtocolVariant allows to specify mode in which control-plane exchange configuration with the sidecar.
+	// Available values are: DELTA_GRPC and GRPC
+	KumaXdsTransportProtocolVariant = "kuma.io/xds-transport-protocol-variant"
```

Comment:
Did you miss the webhook validation for this annotation's value?

Reply:
I was reviewing the code and noticed that we don't currently validate all annotation values. I've updated the code to throw an error during Dataplane conversion if an invalid value is found